src/Globe/TileMesh.js

```diff
@@ -72,28 +63,6 @@ TileMesh.prototype = Object.create(NodeMesh.prototype);
 
 TileMesh.prototype.constructor = TileMesh;
 
-TileMesh.prototype.buildHelper = function buildHelper() {
```

Comment:
I'm anticipating #239 where the helpers are built in a debug layer. See [here](https://github.com/iTowns/itowns2/blob/c179f064324f18f73c4b1e23ded7250a9978d4f8/src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js#L370).